__tests__/integration.test.js

```diff
@@ -253,3 +253,28 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/comments/:comment_id", () => {
+  test("DELETE:204 deletes the given comment by comment_id", () => {
+    return request(app)
+      .delete("/api/comments/3")
+      .expect(204)
+      .then(({ body }) => expect(body).toMatchObject({}));
```

Comment:
You can simplify this test and check that its a 204, like this:
`return request(app).delete("/api/comments/3").expect(204);`